local-cli/bundle/bundle.js

```diff
@@ -8,26 +8,30 @@
  */
 
 const buildBundle = require('./buildBundle');
-const bundleCommandLineArgs = require('./bundleCommandLineArgs');
-const parseCommandLine = require('../util/parseCommandLine');
 const outputBundle = require('./output/bundle');
 const outputPrepack = require('./output/prepack');
+const bundleCommandLineArgs = require('./bundleCommandLineArgs');
 
 /**
  * Builds the bundle starting to look for dependencies at the given entry path.
  */
-function bundleWithOutput(argv, config, output, packagerInstance) {
-  const args = parseCommandLine(bundleCommandLineArgs, argv);
+function bundleWithOutput(argv, config, args, output, packagerInstance) {
   if (!output) {
     output = args.prepack ? outputPrepack : outputBundle;
   }
   return buildBundle(args, config, output, packagerInstance);
-
 }
 
-function bundle(argv, config, packagerInstance) {
```

Comment:
I think it is an array of keys and values mixed, the way argv does it.
They got just serialised with a comma.


On Thursday, 28 July 2016, Mike Grabowski <notifications@github.com> wrote:

> In local-cli/bundle/bundle.js
> <https://github.com/facebook/react-native/pull/7899#discussion_r72574636>:
>
> >  }
> >
> > -function bundle(argv, config, packagerInstance) {
>
> Why command line args are comma separated? Shouldn't be --entry-file
> /path/SomeAppBundle.js --platform ios etc?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/react-native/pull/7899/files/a6859e7e47a8d5d4ce0802579c2df318295a5e08#r72574636>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACBdWB2WEHGVjx70-03HPq1z0RFZBi2bks5qaFg_gaJpZM4Is8X3>
> .
>